[
 {
  "author": "rding", 
  "date": "2011-05-19T23:59:24.300596Z", 
  "msg": [
   "Replace RuntimeException with ParserException"
  ], 
  "revision_id": "1125185"
 }, 
 {
  "author": "trawick", 
  "date": "2011-05-19T23:59:23.533261Z", 
  "msg": [
   "mention the more prominent of the recent apr_fnmatch() tweaks"
  ], 
  "revision_id": "1125184"
 }, 
 {
  "author": "rding", 
  "date": "2011-05-19T23:57:26.021824Z", 
  "msg": [
   "Replace RuntimeException with ParserException"
  ], 
  "revision_id": "1125183"
 }, 
 {
  "author": "rhuijben", 
  "date": "2011-05-19T23:47:27.203311Z", 
  "msg": [
   "* subversion/tests/cmdline/special_tests.py", 
   "(replace_symlink_with_dir): Following up on r1125177, an attempt to fix this", 
   "test on systems that do support symlinks."
  ], 
  "revision_id": "1125182"
 }, 
 {
  "author": "stack", 
  "date": "2011-05-19T23:39:40.171090Z", 
  "msg": [
   "[maven-release-plugin] prepare for next development iteration"
  ], 
  "revision_id": "1125181"
 }
]